arrayOperationTask/script.js

```diff
@@ -0,0 +1,33 @@
+// reverse
+const reverseFun = (data)=>{
+   let reverserArr = []

```

Comment:
This is correct @eligifty-coder and absolutely works. However, is there something else we could do to make this more optimal?

Right now we are using another array called `reverseArr`. Is there a way we could reverse this array in-place using the same original array, without having to use another array?

Let me know what you think.

Reply:
let me give it some thought

Reply:
I came up with something and have updated the code appropriately